### PR TITLE
Marcos.reginato/testing fabs

### PR DESCRIPTION
**Pull request by regmarcos**
Hi guys! Now the button's methods are tested too. As you can see the class has the 100% coverage.

Here is the evidence
coverage pr7
tests pr7

### CODE REVIEW

File: app/src/test/java/com/globant/myapplication/CharacterViewModelTest.kt [hunk 1]
```diff
@@ -50,7 +50,8 @@ class CharacterViewModelTest : AutoCloseKoinTest() {
     @Before
     fun setup() {
         Dispatchers.setMain(mainThreadSurrogate)
-        declareMock<GetCharacterByIdUseCase>()
+        //declareMock<GetCharacterByIdUseCase>()
+        //declareMock<GetRepositoryUseCase>()
```
Comment (nicolbidone) [inline app/src/test/java/com/globant/myapplication/CharacterViewModelTest.kt]: pls delete commented lines

**Review by nicolbidone — CHANGES_REQUESTED** [review]

File: app/src/test/java/com/globant/myapplication/CharacterViewModelTest.kt (hunk 1 above)
Comment (regmarcos) [reply app/src/test/java/com/globant/myapplication/CharacterViewModelTest.kt]: done

**Review by SartiMau — DISMISSED** [review]

**Review by nicolbidone — APPROVED** [review]

File: app/src/test/java/com/globant/viewmodels/RecyclerCharactersViewModelTest.kt [hunk 2]
```diff
@@ -25,6 +24,7 @@ import org.junit.Rule
 import org.junit.Test
 import org.koin.test.AutoCloseKoinTest
 
+
```
Comment (BQuinteros) [inline app/src/test/java/com/globant/viewmodels/RecyclerCharactersViewModelTest.kt]: Remove extra space pls

Comment (regmarcos) [reply app/src/test/java/com/globant/viewmodels/RecyclerCharactersViewModelTest.kt]: done

**Review by BQuinteros — APPROVED** [review]

**Review by nicolbidone — APPROVED** [review]

**Review by SartiMau — APPROVED** [review]

**Pull request merged**